README.md

```diff
@@ -57,12 +57,38 @@ This project supports a basic [Vagrant](http://docs.vagrantup.com/v2/getting-sta
 
 1. Run vagrant up command
 
-    vagrant up
+        vagrant up
+
 
 2. Visit [http://localhost:8085](http://localhost:8085) in your browser
 
 Look in [Vagrantfile](Vagrantfile) for configuration details.
 
+### Docker server
+
+This project supports a basic [Docker](https://www.docker.com/whatisdocker/) configuration
+
+1. [Install Docker](http://docs.docker.com/installation/)
+2. `cd` into the directory of the project
+3. Build the docker image based on the *Dockerfile*:
+        
+        docker build .
```

Comment:
I also prefer fenced code blocks, however this file already uses indenting, so best not to mix styles.

FWIW, the syntax highlighting is a feature of [GitHub Flavored Markdown](https://help.github.com/articles/github-flavored-markdown/), so it does not work everywhere, although fenced code blocks do, as they are just a style of ordinary back tic use.

I'd rather see all the code blocks fenced, if we're going to stay with indenting, the number of leading spaces indented on the one-liners is inconsistent. Standard Markdown converts text with four spaces at the beginning of each line into a code block. For some reason we have some one-line code blocks in this file with four and some with more.